vendor-onboarding-checklist.md

```diff
@@ -0,0 +1,175 @@
+# Vendor onboarding checklist
+
+## Purpose
+
+This document supports the Division of Public Assistance Eligibility
+Information System Replacement (EIS-R) Project. Specifically, this
+document addresses our agile acquisition practice and onboarding new
+vendor staff. Within agile acquisition, we anticipate bringing in a
+potentially different vendor every three to four months, possibly more
+often when we begin running multiple acquisitions simultaneously. With
+short timelines, this onboarding checklist will bring new vendor staff
+up to speed as quickly as possible.
+
+# Audience
+
+This document intends to inform EIS-R State of Alaska project staff and
+the vendors we engage with in various product-increment acquisitions.
+
+# Roles
+
+This section identifies the anticipated vendor roles that will be
+engaged with the
+project.
+
+| **Role**                 | **Description**                                                                             |
+| ------------------------ | ------------------------------------------------------------------------------------------- |
+| Project manager          | Manage the project                                                                          |
+| Technical lead           | Lead the technical work                                                                     |
+| Developers               | Develop client and server application components using CSS, HTML5, Javascript and .NET Core |
+| Researcher               |                                                                                             |
+| User experience designer | Design the user experience                                                                  |
+| Visual designer          |                                                                                             |
+| Scrum Master             | Lead internal daily scrum sessions                                                          |
+
+# Tools and resources
+
+This section identifies the tools and resources that vendors will need
+access to work with and what type of access they will need.
+
+<table>
+<thead>
+<tr class="header">
+<th><strong>Role</strong></th>
+<th><strong>Tools and Resources</strong></th>
+<th><strong>Able to</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>Project manager</td>
+<td><p>State of Alaska LDAP account</p>
+<p>State of Alaska VPN</p>
+<p>DHSS domain account</p>
+<p>DHSS SharePoint</p>
+<p>Visual Studio Team Services</p>
+<p>Trello</p>
+<p>Slack</p>
+<p>Alaska DHSS GitHub</p>
+<p>Mural</p></td>
+<td><ul>
+<li><p>Access Alaska WAN</p></li>
+<li><p>Access DHSS domain secured resources</p></li>
+<li><p>CRU docs on SharePoint</p></li>
+<li><p>Read and update work-items and PR comments in VSTS</p></li>
+<li><p>Read and update Trello</p></li>

```

Comment:
I think it's good for us to leave in tools like Trello that we currently use but might not in the future.